src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,81 @@
+import { useParams, useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { PersonSex } from '../types/PersonSex';
+import { getSearchWith } from '../utils/searchHelper';
+import { filterPeople } from '../utils/filterPeople';
+import { sortPeople } from '../utils/sortPeople';
+
+type Props = {
+  people: Person[]
+  isLoading: boolean,
+  isError: boolean,
+};
+
+export const PeoplePage: React.FC<Props> = ({ people, isLoading, isError }) => {
+  const { personSlug = '' } = useParams();
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') as PersonSex || PersonSex.All;
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const sort = searchParams.get('sort') || '';
```

Comment:
Replace the '||' operator with '??' operator to properly handle default value for 'sort'.